docs/content/any/project/changelogs/2021-11-09.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Release 2021-11-09
+menuTitle: "2021-11-09"
+any: true
+description: >-
+  Changelog for Release 2021-11-09 (0.23.0) containing new features,
+  bug fixes, and more.
+---
+
+## `oso` 0.23.0
+
+### Core
+
+#### Breaking changes
+
+{{% callout "Warning" "orange" %}}
+  This release contains breaking changes. Be sure to follow migration steps
+  before upgrading.
+{{% /callout %}}
+
+Data filtering now treats accessing an undeclared field as an error. All fields
+used in data filtering queries must now be registered ahead of time by including
+them in the `fields` parameter of the `register_class` Oso API function, which
+previously was only necessary for data relations. For example, to use data filtering
+with the rule
+
+```polar
+allow(user, _, foo: Foo) if foo.user_name = user.name;
+```
+
+`user_name` must be included in the `register_class` call for `Foo`:
+
+```py
+# an example in Python
+oso.register_class(Foo, fields={'user_name': str })
+```
+
+This change *only* affects data filtering. Other Oso APIs require no new configuration.
+
+#### Other bugs & improvements
+
+- Fixed a bug where a negated constraint on a dot lookup could cause Polar to crash
+  when the underlying variable became bound.
+- Removed syntax for parenthesized specializers like `f(_: (x));`, which don't
+  currently achieve anything.
+
+### Python
+
+#### Other bugs & improvements
+- Thanks to [Clara McCreery](https://github.com/chmccreery) for a correction to our
+  Python data filtering docs!
+
+#### Platform support
+
+We now publish wheels for musl-based Linux distributions (through the `musllinux`
+tag) and for ARM-based MacOS systems (through the `macosx_11_0_arm64` tag).
+We now publish wheels for Python 3.10 for all supported platforms.
+
+On those systems, you should now be able to use `pip install oso` to get the
+latest Oso package.
+
+### Node.js
+
+#### New features
+- It is now possible to customize the way that types are determined. By passing
```

Comment:
nit: usually each new feature goes under a `#####` heading instead of as part of an unordered list